pkg/liqoctl/status/remoteinfo_test.go

```diff
@@ -0,0 +1,272 @@
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	netv1 "github.com/liqotech/liqo/apis/net/v1alpha1"
+)
+
+var _ = Describe("Remoteinfo", func() {
+	const (
+		rootTitle    = "Remote Clusters Information"
+		namespace    = "liqo"
+		clusterID1   = "e126183a-5445-404f-8802-fdd5ed64b4ec"
+		clusterName1 = "cluster1"
+		clusterID2   = "d945afca-3a15-45ef-a472-41f1803592d1"
+		clusterName2 = "cluster2"
+	)
+	var (
+		clientBuilder fake.ClientBuilder
+		rootNode      = newRootInfoNode(rootTitle)
+		ric           *RemoteInfoChecker
+	)
+
+	BeforeEach(func() {
+		ctx = context.Background()
+		_ = netv1.AddToScheme(scheme.Scheme)
+		clientBuilder = *fake.NewClientBuilder().WithScheme(scheme.Scheme)
+	})
+
+	Context("Creating a new remoteInfoChecker", func() {
+		JustBeforeEach(func() {
+			ric = newRemoteInfoChecker(namespace, &[]string{}, &[]string{}, clientBuilder.Build())
+		})
+		It("should return a valid remoteInfoChecker", func() {
+			ricTest := &RemoteInfoChecker{
+				client:             clientBuilder.Build(),
+				namespace:          namespace,
+				clusterNameFilter:  &[]string{},
+				clusterIDFilter:    &[]string{},
+				errors:             false,
+				collectionErrors:   nil,
+				rootRemoteInfoNode: rootNode,
+			}
+			Expect(*ric).To(Equal(*ricTest))
+		})
+	})
+
+	Context("Checking if a cluster ID or a cluster name are included between filters", func() {
+		Context("Checking cluster ID", func() {
+			BeforeEach(func() {
+				ric.clusterIDFilter = &[]string{"cluster-id-1", "cluster-id-2"}
+			})
+			When("Cluster ID is included", func() {
+				It("should return true", func() {
+					Expect(ric.argsFilterCheck("cluster-id-1", "")).To(BeTrue())
+				})
+			})
+			When("Cluster ID is not included", func() {
+				It("should return false", func() {
+					Expect(ric.argsFilterCheck("cluster-id-3", "")).To(BeFalse())
+				})
+
+			})
+		})
+		Context("Checking cluster name", func() {
+			BeforeEach(func() {
+				ric.clusterNameFilter = &[]string{"cluster-name-1", "cluster-name-2"}
+			})
+			When("Cluster name is included", func() {
+				It("should return true", func() {
+					Expect(ric.argsFilterCheck("", "cluster-name-1")).To(BeTrue())
+				})
+			})
+			When("Cluster name is not included", func() {
+				It("should return false", func() {
+					Expect(ric.argsFilterCheck("", "cluster-name-3")).To(BeFalse())
+				})
+
+			})
+		})
+
+	})
+
+	Context("Getting Collect() result", func() {
+		var infoNode InfoNode
+		BeforeEach(func() {
+			clientBuilder.WithObjects(
+				&v1.ConfigMap{
+					TypeMeta: metav1.TypeMeta{
+						Kind: "ConfigMap",
+					},
+					ObjectMeta: metav1.ObjectMeta{
+						Labels: map[string]string{
+							"app.kubernetes.io/name": "clusterid-configmap",
+						},
+						Namespace: namespace,
+					},
+					Immutable: new(bool),
+					Data: map[string]string{
+						"CLUSTER_ID":   clusterID1,
+						"CLUSTER_NAME": clusterName1,
+					},
+					BinaryData: map[string][]byte{},
```

Comment:
Same here (and below)

Reply:
I think would be a better idea to rewrite tests, because a lot of things have changed